### PR TITLE
wpd: add package

### PR DESCRIPTION
**Pull request by kleo**
Add package

wpd - WiFi Password Display for Android Oreo+ (Root)

Package description: Shows all WiFi networks and passwords stored on your phone
License: GPL v3.0

Usage: `sudo wpd`

### CODE REVIEW

File: packages/wpd/build.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/kbeflo/wpd
+TERMUX_PKG_DESCRIPTION="Shows all WiFi networks and passwords stored on your phone"
```
Comment (Grimler91) [inline packages/wpd/build.sh]: Please indicate in the description that oreo or newer is required

**Comment by Grimler91** [comment]
If I make a wifi with password `a!@#$%^&*()_+~}{|"][:;<>?,./` and run wpd it shows the password: `a!@#$%^&amp;*()_+~}{|`. If you use `xml` instead of `grep` + `sed` then it should replace all the special characters for you so that you don't have to worry about it

**Comment by Grimler91** [comment]
This also fails for wifis with spaces in the WiFi name, they are splitted when looping to display the results. 

Also, if you have some wifis without a PSK, or if it uses some other security than WPA2, the returned lists will be shifted as the number of wifis is larger than the number of passwords

**Comment by Jc-sims** [comment]
git submodule init when it asks for user name and password

**Comment by Grimler91** [comment]
Hi, sorry for the delay. 

I still get lists that are out-of-sync or incorrect. The last 4 wifis in my list are on this format:

```
Bageri - NONE
Creperi_Guests - h33h2gagjhrte
.Oresundstag_Wi-Fi - 
iPhone - 
```

But it should be

```
Bageri - NONE
Creperi_Guests - NONE
.Oresundstag_Wi-Fi - NONE
iPhone - h33h2gagjhrte
```

For some reason the sed patterns are skipping the password of Creperi_Guests and .Oresundstag_Wi-Fi

**Comment by g3ymer** [comment]
> Add package
> 
> wpd - WiFi Password Display for Android Oreo+ (Root)
> 
> Package description: Shows all WiFi networks and passwords stored on your phone
> License: GPL v3.0
> 
> Usage: `sudo wpd`

**Comment by Grimler91** [comment]
Closing since this have not seen any activity since last year

**Pull request closed without merging**